source/custom/nirfsg_restricted_service.custom.cpp

```diff
@@ -0,0 +1,19 @@
+#include <nirfsg_restricted/nirfsg_restricted_service.h>
+
+namespace nirfsg_restricted_grpc {
+
+::grpc::Status NiRFSGRestrictedService::ConvertApiErrorStatusForViSession(::grpc::ServerContextBase* context, int32_t status, ViSession vi)
```

Comment:
Is there a known error case that you can test to exercise this custom code?

Reply:
No. I refer to this PR https://github.com/ni/grpc-device/pull/830 on adding niscope restricted component. I feel they were similar.